client-js/app/elements/labrad-nodes.ts

```diff
@@ -331,152 +392,227 @@ export class LabradNodes extends polymer.Base {
   }
 
 
-  /**
-   * Find the index in the node info list of a node with the given name.
-   *
-   * If no such node is found, returns -1.
```

Comment:
Maybe we could name these `indexOfServer` and `indexOfNode`, then clear they work like `indexOf`, as you say. Not a big deal either way.